src/app/parcels/form/ParcelForm.tsx

```diff
@@ -219,8 +220,7 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
 
         const isDelivery = fields.shippingMethod === "Delivery";
 
-        const formToAdd = {
-            primary_key: parcelId!,
+        const parcelRecord = {
```

Comment:
Renamed so it's the same name as the variable in clientDatabaseFunctions. Primary key removed as it wasn't used I think..

Reply:
Yeah this is good, I don't think the primary key was being used either